roottest/root/tree/cloning/CMakeLists.txt

```diff
@@ -1,75 +1,23 @@
-if(ROOTTEST_DIR)
-      set(ROOT_EVENT_DIR ${ROOTTEST_DIR}/root/treeformula/event/)
-else()
-      set(ROOT_EVENT_DIR ${ROOT_SOURCE_DIR}/roottest/root/treeformula/event/)
-endif()
-
 # Generating dataset from roottest-treeformula-event-make test
 # FIXME: it will be nice to move roottest-treeformula-event to CMake and add it as dependency
 # To fix runtime_cxxmodules, we need to use already build artefacts.
 
-if(TARGET onepcm)
-      set(EventDependencies "onepcm")
-endif()
-
-if(NOT TARGET eventexe)
-      ROOT_GENERATE_DICTIONARY(EventDict ${ROOT_EVENT_DIR}/Event.h
-                  LINKDEF ${ROOT_EVENT_DIR}/EventLinkDef.h)
-
-      ROOTTEST_LINKER_LIBRARY(EventTreeFormula TEST ${ROOT_EVENT_DIR}/Event.cxx EventDict.cxx
-                  LIBRARIES ROOT::Core ROOT::Tree ROOT::Hist ROOT::MathCore)
-
-      ROOTTEST_GENERATE_EXECUTABLE(EventGenerate ${ROOT_EVENT_DIR}/MainEvent.cxx
-                  LIBRARIES ROOT::Core ROOT::RIO ROOT::Net ROOT::Tree ROOT::Hist ROOT::MathCore EventTreeFormula)
-
-      if(MSVC)
-        add_dependencies(EventGenerate EventTreeFormula)
-      endif()
-
-      add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/Event.root
-                  COMMAND ./EventGenerate 6 0 0 1 30 > log
-                  DEPENDS EventGenerate ${EventDependencies})
-
-      add_custom_target(event-generation ALL DEPENDS Event.root)
-
-      add_custom_command(
-        TARGET event-generation
-        COMMAND ${CMAKE_COMMAND} -E copy
-                Event.root
-                event1.root
-        COMMAND ${CMAKE_COMMAND} -E copy
-                Event.root
-                event2.root)
```

Comment:
Since `eventexe` is now always target, we could also have replace the `generate-eventfile.sh` with those (adapted to the current/new location) and thus enabling the test on Windows.

Reply:
Great idea! I'll get there step by step :slightly_smiling_face: